### DISCUSSION TITLE
missing .font-myfont directive in generated style.css when using custom font

### DISCUSSION THREAD

**Discussion by sandeep-gh** [tailwindlabs/tailwindcss#13486] [Help] [answered]
My setup is as follows: 
#. In tailwind.config.ts
```
theme: {
    extend: {
fontFamily: {
		   'poppins': ['Poppins', 'sans-serif'],
	   },
}
}
````
#. In src/app.postcss
```
@layer base {
  @font-face {
    font-family: 'Poppins';
    font-style: normal;
    font-weight: 400;
    font-display: swap;
    src: url(/fonts/Poppins.woff2) format('woff2');
  }
}
```
However, when I do a vite build, I don't see the css directive ".font-poppins" in the generated CSS file. I am not sure how to debug that or how to fix it.

**Comment by alex-krasikau** [comment, marked as answer]
Hey @sandeep-gh!

Ensure you have used the `font-poppins` class in your code for Tailwind CSS to generate the CSS declaration. Typically, apply the `font-poppins` class to your `body` element:
```html
<html>
  <body class="font-poppins">
  ...
  </body>
</html>
```

After adding this, you will find the `.font-poppins` declaration in the CSS file. Check out an example on [Play Tailwind](https://play.tailwindcss.com/orJz0uERab). 

![Screenshot](https://github.com/tailwindlabs/tailwindcss/assets/97882909/4d0ad0ba-4519-4635-88de-731c680c5171)
<img width="604" alt="CleanShot 2024-04-09 at 19 19 57@2x" src="https://github.com/tailwindlabs/tailwindcss/assets/97882909/0a970c73-9458-420e-b111-16a77926827d">

**Reply by sandeep-gh** [reply]
This "Ensure you have used the font-poppins class in your code for Tailwind CSS to generate the CSS declaration." was the problem.  Thank you so much.
